src/content/introduction.md

```diff
@@ -8,3 +8,91 @@
 
 >Rob Brander - @rbrander
 
+---
+
+## Prerequisites
+
+- Access to the command line
+- Node.js (v6 or higher)
+- NPM (v3 or higher)
+- Code editor (Sublime, Atom, etc.)
+- Some JavaScript knowledge (ES5/ES6)
+- Basic React Knowledge (Part 1 of this course)
+- Basic Redux Knowledge (Part 2 of this course)
+
+---
+
+## Structure of this Training
+
+- 2 hours training
+- 20 min Intro/Slides
+- 1h 40 min Live Coding
+
+---
+
+## Curriculum
+
+- Routing (with React Router V4)
+- Server-Side Rendering (with express.js)
+- TBD
```

Comment:
I think we should either add Lazy Loading here or remove the bullet all together